src/pages/guides/quickstart/index.md

```diff
@@ -100,7 +115,7 @@ The SDK should only be initialized once each page. To initialize the SDK, pass t
 })();
 ```
 
```

Comment:
**_The SDK should only be initialized once on each page._** 

Let's add this as an info block.

Reply:
You're a fan of Info Blocks :-D 
Added!